app/controllers/AnnotationController.scala

```diff
@@ -48,9 +46,11 @@ class AnnotationController @Inject()(annotationDAO: AnnotationDAO,
       annotation <- provider.provideAnnotation(typ, id, request.identity) ?~> "annotation.notFound"
       restrictions <- provider.restrictionsFor(typ, id) ?~> "restrictions.notFound"
       _ <- restrictions.allowAccess(request.identity) ?~> "notAllowed" ~> BAD_REQUEST
-      js <- annotationService.publicWrites(annotation, request.identity, Some(restrictions)) ?~> "annotation.write.failed"
+      js <- annotationService
+        .publicWrites(annotation, request.identity, Some(restrictions)) ?~> "annotation.write.failed"
+      typedType <- AnnotationType.fromString(typ).toFox
       _ <- Fox.runOptional(request.identity) { user =>
-        if (typ == AnnotationType.Task || typ == AnnotationType.Explorational) {
+        if (typedType == AnnotationType.Task || typedType == AnnotationType.Explorational) {
```

Comment:
The changes in the snapshots were necessary because this condition is now checed correctly. Due to this change the annotation snapshot depended on the time between the first and the second call of this route. This time changes from run to run, so it isn't a good idea to check this. The test already implemented a method to replace volatile values, so I added the key `tracingTime` to be replaced and called it in the specific test. This lead to the changes in the annotation snapshot, where other volatile values are now replaced, too and to the changes in the task snapshot. Regarding the now changed values in the annotation snapshot, I added a check for the annotation id and the skeleton id as they're not volatile in this scenario. 
The user-facing doesn't really change except for the fact we may log additional time because of corrected check.